packages/vector_graphics_codec/lib/vector_graphics_codec.dart

```diff
@@ -18,6 +18,9 @@ class VectorGraphicsCodec {
   static const int _cubicToTag = 34;
   static const int _closeTag = 35;
   static const int _finishPathTag = 36;
+  static const int _opacityLayerTag = 37;
+  static const int _saveLayer = 38;
```

Comment:
Here and elsewhere: Why have this in the format? We should only need this for transforms that we don't do anymore.

Reply:
we need a save layer / paint to do opacity groups, no?

Reply:
Sorry, I read this too fast and thought this was a "save".

However - I don't think we should have an opacity layer tag. We should just have saveLayer have a paint on it.

Reply:
Yup agreed, If we do the paint then its just a generic layer anyway. For some reason i thought canvas had a pushOpacity method 🤔